lib/active_fedora/loadable_from_json.rb

```diff
@@ -105,6 +105,30 @@ def reflection(predicate)
       end
     end
 
+    # Adapt attributes read from Solr to possible minor changes in data model
+    # since the attributes were saved.
+    # @param attrs [Hash] attributes read from Solr
+    # @return [Hash] the adapted attributes
+    def adapt_attributes(attrs)
+      new_attributes = {}
+      self.class.attribute_names.each do |a|
+        reflection = self.class.reflect_on_property a
+        new_attributes[a] = if !reflection
```

Comment:
Are lines 116-128 all for setting `new_attributes[a]`? I'd consider either changing the indent to make that obvious or pulling this into a private method of its own for cleaner/leaner separation.